backend/servicing_calendar/servicing_calendar/settings.py

```diff
@@ -0,0 +1,124 @@
+"""
+Django settings for servicing_calendar project.
+
+Generated by 'django-admin startproject' using Django 4.0.6.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.0/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-tcnwa@ofpe9^6ry#_3%zb)mtljy7+a$0q99*quzqy*vb49v7(-'
```

Comment:
Move `SECRET_KEY` into `.env`

Reply:
Thank you to remind me I have already pushed a new version of setting.py to set-up branch please take a look.

Reply:
Sorry I pushed wrong version.

Reply:
Alright then